postgres/map.jinja

```diff
@@ -30,3 +30,8 @@
   {% set pc_cmd = '{0} --encoding={1}'.format(pc_cmd, postgres.cluster.encoding) %}
 {% endif %}
 {% do postgres.update({'prepare_cluster_cmd': pc_cmd}) %}
+
+{# handle selinux enforcing #}
```

Comment:
why not base it on the grains on the system:
```
sudo salt-call grains.item selinux
local:
    ----------
    selinux:
        ----------
        enabled:
            True
        enforced:
            Enforcing
```

Reply:
Thanks. Which grain?

Reply:
@noelmcloughlin : `selinux:enforced`

Reply:
sorry okay, there is selinux grain. good idea.